src/ussd_pmtct.js

```diff
@@ -231,7 +353,7 @@ go.app = function() {
                         return self.im.user
                         .set_lang(self.im.user.answers.contact_identity.details.lang_code || "eng_ZA")
                         .then(function(lang_set_response) {
-                            return self.states.create("state_optout_reason_menu");
+                            return self.states.create("state_optout_reason_menu", identity);
```

Comment:
please save it as a user answer and reuse that rather than passing it through everywhere.